source/django_package/apps.py

```diff
@@ -2,4 +2,4 @@
 
 
 class FunctionsConfig(AppConfig):
-    name = 'django_package'
+    name = 'source.django_package'
```

Comment:
`django_package` is not a good name for this app, unless you have your own idea for the name, it should be `django_sourcerer`.